services/ui-src/src/components/pages/ReviewSubmit/ReviewSubmitPage.tsx

```diff
@@ -63,6 +62,15 @@ export const ReviewSubmitPage = () => {
     setHasError(!!document.querySelector("img[alt='Error notification']"));
   }, [fetchReport]);
 
+  useEffect(() => {
+    setIsPermittedToSubmit(
+      (userIsEndUser &&
+        report?.status === ReportStatus.IN_PROGRESS &&
+        !hasError) ||
+        false
+    );
+  }, [userIsEndUser, report?.status, hasError]);
+
```

Comment:
Can leave this in or take it out - See note about my findings here: https://qmacbis.atlassian.net/browse/MDCT-2661